SWIPs/swip-message_to_honey.md

```diff
@@ -26,42 +25,75 @@ This SWIP is part of a series of SWIPs (but can be implemented on it's own). To
 
 ## Motivation
 <!--The motivation is critical for SWIPs that want to change the Swarm protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the SWIP solves. SWIP submissions without sufficient motivation may be rejected outright.-->
-In Swarm, nodes send various types of messages; chunk requests, chunk delivery, PSS… to name a few. Every message is priced differently to reflect the different load such messages have on the system. To ensure optimal usage and health of the network it is essential that these prices are quoted relative to the load they induce on the network. If this is not the case certain messages may be too cheap - resulting in unprofitability for service providers, while others may be too expensive - causing users to avoid the service. The Swarm network must be used by real nodes to get insights on how messages should be priced relative to each other. Because of this, there is a need for an update process of the relative prices of messages on a live network. This update process should be atomic: either all nodes upgrade or none, which is currently not facilitated by the hardcoded values in the Swarm source code. Therefore, if in the current implementation prices change, nodes running different releases will account for messages differently. Eventually, this will result in disconnects due to balance differences between nodes. This SWIP proposes an on-chain price oracle to enable an atomic update process of message prices without requiring nodes to update their source-code. 
+In Swarm, nodes send various types of messages; chunk requests, chunk delivery, PSS… to name a few. Every message is priced differently to reflect the different load such messages have on the system. To ensure optimal usage and health of the network it is essential that these prices are quoted relative to the load they induce on the network. If this is not the case certain messages may be too cheap - resulting in unprofitability for service providers, while others may be too expensive - causing users to avoid the service. The Swarm network must be used by real nodes to get insights on how messages should be priced relative to each other. Hence, there is a need for an update process of the relative message price on a live network. This update process should be atomic: either all nodes upgrade or none, which is currently not facilitated by the hardcoded values in the Swarm source code. Therefore, if in the current implementation prices change, nodes running different releases will account for messages differently. Eventually, this will result in disconnects due to balance differences between nodes. This SWIP proposes an on-chain price oracle to enable an atomic update process of message prices.
```

Comment:
use `—` (em dash) or ` – ` (spaced en dash) instead of hyphens to mark breaks in sentences.

Reply:
https://getitwriteonline.com/articles/en-dashes-em-dashes/

Reply:
Fixed